### DISCUSSION TITLE
YOLOv5 version 6.1 architecture

### DISCUSSION THREAD

**Discussion by fpearmaine3** [ultralytics/yolov5#7160] [Show and tell]
Hello all.
Attached is my interpretation of the architecture that has changed since the first release of YOLOv5, I think it's quite nice and accurate. Please let me know if it's incorrect in any way.
Thank you all

<img width="998" alt="YOLO_arch" src="https://user-images.githubusercontent.com/33327952/160244223-91058411-176d-4208-90d5-f598399bb55b.png">
.

**Comment by nrupatunga** [comment]
Thank you for nice visualisation of the architecture and if I'm not wrong, the following blocks in the head section need to be swapped, and the sequence of upsampling by a factor of 2 indicates the same

![image](https://user-images.githubusercontent.com/980580/162230164-a7c25f9d-3de9-45b0-a2d6-7f8607052d03.png)

**Comment by liamh1999** [comment]
I think there is a conv layer missing in the neck between the c3 and upsample ?
